Todoey/Controllers/ToDoListViewController.swift

```diff
@@ -40,7 +43,10 @@ class ToDoListViewController: UITableViewController {
 	}
 
 	override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-		itemArray[indexPath.row].done.toggle()
+		let row = indexPath.row
+		//		context.delete(itemArray[row])
+		//		itemArray.remove(at: row)
```

Comment:
⛏️  <- Stands for "nitpick", optional change

```suggestion
                //       Keeping this in case deletion is implemented later
		//		context.delete(itemArray[row])
		//		itemArray.remove(at: row)
```